rfcs/sliders-component.md

```diff
@@ -0,0 +1,122 @@
+# Sliders Component
```

Comment:
Why use "Sliders" instead of "Slider"? Seems like the latter makes more sense and is the more common choice among other frameworks.

Reply:
fixed